src/modules/add-remove.js

```diff
@@ -0,0 +1,94 @@
+import { tasks, save } from './storage.js';
+
+const removeTodo = (button) => {
+  button.parentElement.remove();
+};
+
+const render = (tasks) => {
+  tasks.forEach((task) => {
+    const listItem = document.createElement('li');
```

Comment:
**[Optional]**

- [ ] Your application is not working properly because the methods which are being tested are not being used in the application. Kindly use these functions in your application to make it work as expected 😊